assets/src/edit-story/app/media/useUploadMedia/useUploadMedia.js

```diff
@@ -89,12 +89,21 @@ function useUploadMedia({ media, setMedia }) {
 
       try {
         const uploadedFiles = await Promise.all(
-          localFiles.map(async (localFile) => ({
-            ...localFile,
-            fileUploaded: getResourceFromAttachment(
+          localFiles.map(async (localFile) => {
```

Comment:
I didn't add `trackEvent`, as I didn't see the point. Felt like I would be repeating events. But I can add this if required.

Reply:
They're slightly different in analytics, so please also add `trackEvent` and include the mime type as I described in the ticket.

I want to have both first, then I can see in analytics how well it works and whether it's indeed redundant or not.

Reply:
Fair enough. Added a fix.